vllm/entrypoints/openai/api_server.py

```diff
@@ -60,6 +59,11 @@
 from vllm.utils import FlexibleArgumentParser, get_open_zmq_ipc_path
 from vllm.version import __version__ as VLLM_VERSION
 
+if envs.VLLM_USE_V1:
+    from vllm.v1.engine.async_llm import AsyncLLMEngine  # type: ignore
+else:
+    from vllm.engine.async_llm_engine import AsyncLLMEngine  # type: ignore
```

Comment:
This won't work with tests, anything that tries to monkeypatch with `m.setenv("VLLM_USE_V1", True)` won't take effect because this runs the check at import time